.circleci/config.yml

```diff
@@ -39,8 +39,8 @@ jobs:
             && rm -rf /var/cache/dnf
 
           # install jdk
-          curl -L \
-           https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.9%2B9/OpenJDK17U-jdk_x64_linux_hotspot_17.0.9_9.tar.gz -o /tmp/jdk.tar.gz
+          curl -Lf \
+           https://github.com/adoptium/temurin17-binaries/releases/download/jdk-jdk-17.0.9%2B9.1/OpenJDK17U-jdk_x64_linux_hotspot_17.0.9_9.tar.gz -o /tmp/jdk.tar.gz
```

Comment:
These new URLs are returning 404 for me.

Reply:
Good catch, fixed